doc/refman/RefMan-syn.tex

```diff
@@ -925,38 +936,40 @@ \subsection{Interpretation scopes used in the standard library of {\Coq}}
 
 \subsubsection{\tt type\_scope}
 
-This includes infix {\tt *} for product types and infix {\tt +} for
-sum types. It is delimited by key {\tt type}.
+This scope includes infix {\tt *} for product types and infix {\tt +} for
+sum types. It is delimited by key {\tt type}, and bound to the class 
+{\tt Sortclass}, which is used as an interpretation scope for certain
+types within an {\tt Ltac} goal match.
```

Comment:
I feel that this last line (from `which`) should be either removed or expanded. It repeats part of what is in section "Binding types of arguments to an interpretation scope" but without being exhaustive. So maybe simply to remove?